### PR TITLE
[DEV-6881] DOD disclosure (part 1)

### PR DESCRIPTION
**Pull request by ebdabbs**
**High level description:**

Adds a DOD 90 day delay disclosure note to the COVID profile, Spending Explorer, and Custom Award Download pages.
(Federal Account profile has a backend dependency and will be implemented in part 2)

**Technical details:**

Also makes it easier to link to specific sections of the About page using query params.

**JIRA Ticket:**
[DEV-6881](https://federal-spending-transparency.atlassian.net/browse/DEV-6881)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Updated unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations

Reviewer(s):
- [x] Design review complete (resolve question about whether the note should go above or below pagination for COVID profile tables)
- [x] Code review complete

### CODE REVIEW

File: src/js/components/about/AboutContent.jsx [hunk 1]
```diff
@@ -64,12 +63,15 @@ const aboutSections = [
...
-const propTypes = {
-    location: PropTypes.object // Router location object
-};
+const AboutContent = () => {
+    const location = useLocation();
+    const history = useHistory();
+    const useQuery = (() => (
+        new URLSearchParams(location.search)
+    ));
+    const query = useQuery();
```
Comment (MaxwellKendall) [inline src/js/components/about/AboutContent.jsx:72]: Can we use this guy instead? https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/helpers/queryParams.js#L3

File: src/js/components/sharedComponents/Note.jsx [hunk 2]
```diff
@@ -16,11 +17,19 @@ export default class Note extends Component {
...
     }
 }
 
 Note.propTypes = propTypes;
+
+export const dodNote = (
+    <>
+        There is a 90 day delay in displaying contract award data, subcontract data,
+        and Account Breakdown by Award (File C) data for the Department of Defense (DOD).
+        For more information, visit our <Link to="/about?section=data-quality">About Page</Link>.
```
Comment (MaxwellKendall) [inline src/js/components/sharedComponents/Note.jsx:33]: Yesssss

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]
Can we reuse the helper?

File: src/js/components/about/AboutContent.jsx (hunk 1 above)
Comment (ebdabbs) [reply src/js/components/about/AboutContent.jsx:72]: Yes, good call!

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**